src/topp/CometAdapter.cpp

```diff
@@ -319,7 +362,8 @@ class TOPPCometAdapter :
     os << "mass_type_parent = " << 1 << "\n";                    // 0=average masses, 1=monoisotopic masses
     os << "mass_type_fragment = " << 1 << "\n";                  // 0=average masses, 1=monoisotopic masses
     os << "precursor_tolerance_type = " << 1 << "\n";            // 0=MH+ (default), 1=precursor m/z; only valid for amu/mmu tolerances
-    os << "isotope_error = " << isotope_error[getStringOption_(Constants::UserParam::ISOTOPE_ERROR)] << "\n";                   // 0=off, 1=0/1 (C13 error), 2=0/1/2, 3=0/1/2/3, 4=-8/-4/0/4/8 (for +4/+8 labeling)
+    os << "isotope_error = " << isotope_error[getStringOption_(Constants::UserParam::ISOTOPE_ERROR)] << "\n";                   
```

Comment:
```suggestion
    os << "isotope_error = " << std::distance(isotope_error.begin(), std::find(isotope_error.begin(), isotope_error.end(), getStringOption_(Constants::UserParam::ISOTOPE_ERROR))) << "\n";                   
```

Reply:
This should probably be isotope_error_str.begin() and isotope_error_str.end(), right?